packages/aeproject-cli/aeproject-init/constants.json

```diff
@@ -26,5 +26,6 @@
 	"dockerNodeYmlFileDestination": "./docker-compose.yml",
 	"dockerCompilerYmlFileDestination": "./docker-compose.compiler.yml",
 	"gitIgnoreFile": ".gitignore",
```

Comment:
do we still need this file ?

Reply:
yes, this is used as title of newly created file